src/middlewared/middlewared/plugins/iscsi_/targets.py

```diff
@@ -97,6 +127,18 @@ async def extend(self, data):
             ),
         ]),
         List('auth_networks', items=[IPAddr('ip', network=True)]),
+        Dict(
+            'options',
+            Str('DataDigest', enum=DigestEnum.choices()),
```

Comment:
I don't think we typically have camelcase parameters. Convention is all lower-case with underscores as-needed.

Reply:
I understand, but we're exposing **raw** iSCSI settings, so IMO this is clearer.

e.g. https://www.rfc-editor.org/rfc/rfc7143#section-13.1

Reply:
@yocalebo do you have strong opinion on this one?